src/snowflake/snowpark/context.py

```diff
@@ -4,10 +4,22 @@
 #
 
 """Context module for Snowpark."""
+from typing import Callable, Optional
+
 import snowflake.snowpark
 
 _use_scoped_temp_objects = True
 
+# This is an internal-only function, used to interrupt registration of UDxF or SPROC objects with Snowflake.
+_internal_only_interrupt_registration: Optional[Callable] = None
+
+# This is an internal-only global flag, used to determine whether to execute code in a client's local sandbox or connect to a Snowflake account.
+_execute_in_local_sandbox: bool = False
+
+
+def get_execute_in_local_sandbox() -> bool:
+    return _execute_in_local_sandbox
```

Comment:
Why not just read the internal flag directly? The getter makes it look like this is a public API, but it's not